lib/tools/example_script.py

```diff
@@ -19,9 +19,9 @@
 Macros:
 API_KEY       -- an Admin API key. It can be found on the admin dashboard page.
 ROOT_URL      -- the root domain of the MarkUs instance.
-ROOT_DIR      -- the directory containing the group repos.
+ROOT_DIR      -- the directory containing the submission repos.
```

Comment:
Do you mean all changes (i.e. leave the script as broken), or just the changes about the repo name (real svn name vs the name exported via the markus' script)?